<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FSM viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="opened_layout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>List of opened FSM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="opened_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="opened_verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="graph">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="graph_fsm">
       <attribute name="title">
        <string>FSM graph</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="graph_fsmScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="graph_fsmScrollAreaWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>565</width>
             <height>578</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="graph_fsmLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="graph_stateCover">
       <attribute name="title">
        <string>State cover graph</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="graph_stateCoverScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="graph_stateCoverScrollAreaWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>565</width>
             <height>578</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="graph_stateCoverLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="graph_transitionCover">
       <attribute name="title">
        <string>Transition cover graph</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QScrollArea" name="graph_transitionCoverScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="graph_transitionCoverScrollAreaWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>565</width>
             <height>578</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="graph_transitionCoverLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="characteristics_layout">
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="characteristics_title">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Fsm characteristics</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_nameLabel">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_name">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_maxNodesLabel">
        <property name="text">
         <string>Number of nodes</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_maxNodes">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_maxInputLabel">
        <property name="text">
         <string>Maximal Input</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_maxInput">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_maxOutputLabel">
        <property name="text">
         <string>Maximal Output</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_maxOutput">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="Line" name="characteristics_line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_isCompletelyDefinedLabel">
        <property name="text">
         <string>is Completely Defined</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_isCompletelyDefined">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_isDeterministicLabel">
        <property name="text">
         <string>is Deterministic</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_isDeterministic">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_isObservableLabel">
        <property name="text">
         <string>is Observable</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="characteristics_isObservable">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="characteristics_isMinimalLabel">
        <property name="text">
         <string>is Minimal</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="characteristics_isMinimal">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Transformation">
    <property name="title">
     <string>&amp;Transformation</string>
    </property>
    <addaction name="actionTransform_to_Observable"/>
    <addaction name="action_Minimise"/>
    <addaction name="actionCalculate_Characterisation_set"/>
    <addaction name="action_Intersection_with_an_other_FSM"/>
   </widget>
   <widget class="QMenu" name="menuTest_cases">
    <property name="title">
     <string>Test cases</string>
    </property>
    <addaction name="actionW_p_Method"/>
    <addaction name="action_W_Method"/>
    <addaction name="action_Create_test_suite"/>
    <addaction name="actionCreate_Output_tree"/>
    <addaction name="actionRun_test_suite_Equivalence"/>
    <addaction name="actionRun_test_suite_Reduction"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="action_Default_Zoom"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Transformation"/>
   <addaction name="menuTest_cases"/>
   <addaction name="menuZoom"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open a FSM</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save current FSM</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close current FSM</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit FSM Viewer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionTransform_to_Observable">
   <property name="text">
    <string>Transform to &amp;Observable</string>
   </property>
  </action>
  <action name="action_Minimise">
   <property name="text">
    <string>&amp;Minimise</string>
   </property>
  </action>
  <action name="actionCalculate_Characterisation_set">
   <property name="text">
    <string>Calculate &amp;Characterisation set</string>
   </property>
  </action>
  <action name="action_Intersection_with_an_other_FSM">
   <property name="text">
    <string>&amp;Intersection with an other FSM</string>
   </property>
  </action>
  <action name="actionW_p_Method">
   <property name="text">
    <string>W&amp;p Method</string>
   </property>
  </action>
  <action name="action_W_Method">
   <property name="text">
    <string>&amp;W Method</string>
   </property>
  </action>
  <action name="action_Create_test_suite">
   <property name="text">
    <string>&amp;Create test suite</string>
   </property>
  </action>
  <action name="actionCreate_Output_tree">
   <property name="text">
    <string>Create &amp;Output tree</string>
   </property>
  </action>
  <action name="actionRun_test_suite_Reduction">
   <property name="text">
    <string>Compare test suite (&amp;Reduction)</string>
   </property>
  </action>
  <action name="actionRun_test_suite_Equivalence">
   <property name="text">
    <string>Compare test suite (&amp;Equivalence)</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_Default_Zoom">
   <property name="text">
    <string>&amp;Default Zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
